cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -597,6 +597,19 @@ def test_categories_with_zero_values_warn(self):
             ],
         )
 
+    def _test_deprecated_fields(self):
```

Comment:
Remove the initial `_` when https://github.com/chanzuckerberg/single-cell-curation/pull/305 is merged